apps/frontend/src/app/PageCharacter/CharacterDisplay/Tabs/TabOptimize/Components/BuildDisplayItem.tsx

```diff
@@ -344,6 +345,9 @@ function ExcludeButton({ id }: { id: string }) {
         size="small"
       >
         {excluded ? <BlockIcon /> : <ShowChartIcon />}
```

Comment:
move this icon to `starIcon` https://mui.com/material-ui/api/button/#button-prop-startIcon

Reply:
done